src/nvim/quickfix.c

```diff
@@ -224,7 +224,6 @@ static char_u *e_no_more_items = (char_u *)N_("E553: No more items");
 #define IS_QF_STACK(qi)       (qi->qfl_type == QFLT_QUICKFIX)
 #define IS_LL_STACK(qi)       (qi->qfl_type == QFLT_LOCATION)
 #define IS_QF_LIST(qfl)       (qfl->qfl_type == QFLT_QUICKFIX)
-#define IS_LL_LIST(qfl)       (qfl->qfl_type == QFLT_LOCATION)
```

Comment:
I'd keep this for symmetry/completeness/what if vim starts to use it (quickfix.c is one of the files that mostly follows upstream vim)

Reply:
fixed